### PR TITLE
docs: add vue package integration docs [skip ci]

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ part of elastic/apm-agent-rum-js#460

### CODE REVIEW

**Comment by vigneshshanmugam** [comment]
Thanks @bmorelli25 Applied the suggestions :)

File: docs/vue-integration.asciidoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    }
+  },
+  created () {
+    this.getName()
+  }
+  methods: {
+    getName() {
+      let transaction = this.$apm.getCurrenTransaction()
+      if (transaction) {
+        this.name = transaction.name
```
Comment (jahtalab) [inline docs/vue-integration.asciidoc]: Maybe add a more common use-case here, like creating a span side the component

File: docs/vue-integration.asciidoc [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      }
+    }
+  }
+}
+</script>
+----
+
+
+`ApmVuePlugin` uses the Navigation guards functionality provided by the VueRouter and will not work
+with any other router instance provided via the plugin options.
```
Comment (jahtalab) [inline docs/vue-integration.asciidoc]: I think the `will not work with any other router instance provided via the plugin options` part is ambiguous. Can you please explain what you wanted to express here?

Comment (vigneshshanmugam) [reply docs/vue-integration.asciidoc]: I was going for `router` option passed to ApmVuePlugin should be an instance of VueRouter.

**Pull request merged**